src/mme-app/monitor_subscriber.c

```diff
@@ -114,6 +116,24 @@ handle_imsi_list_req(struct monitor_imsi_req *mir, int sock_fd)
     return;
 }
 
+/**
+* Flush Imsis.
+*/
+void
+handle_imsi_flush_req()
+{
+    struct UE_info *head_entry = ue_list_head;
+    log_msg(LOG_DEBUG, "imsi flush request, ue count %d\n", g_total_UE_count);
+    while(head_entry)
+    {
+        release_ue_entry(head_entry);
```

Comment:
Can we send message to UE to reattach..That would really be good in testing

Reply:
Will raise issue for this update.